toqito/channels/choi.py

```diff
@@ -88,7 +88,7 @@ def choi(a_var: int = 1, b_var: int = 1, c_var: int = 0) -> np.ndarray:
 
     See Also
     ==========
-    reduction
+    Reduction: `toqito reduction <https://toqito.readthedocs.io/en/latest/autoapi/channels/reduction/index.html>`_
```

Comment:
This is not what the issue description wants. Please take a look at the screenshot and the linked PR in the issue description.